File: sdkwork-ai-sdk-python/sdkwork_ai_sdk/models/thread_message_response.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class ThreadMessageResponse:
    """OpenAI thread message object"""
    id: str = None
    object: str = None
    role: str = None
    status: str = None
    content: List[ContentBlock] = None
    metadata: Dict[str, str] = None
    created_at: int = None
    thread_id: str = None
    incomplete_details: Any = None
    completed_at: int = None
    incomplete_at: int = None
    assistant_id: str = None
    run_id: str = None
    file_ids: List[str] = None

File: sdkwork-ai-sdk-kotlin/src/main/kotlin/com/sdkwork/ai/ThreadMessageResponse.kt
package com.sdkwork.ai

data class ThreadMessageResponse(
    val id: String? = null,
    val object: String? = null,
    val role: String? = null,
    val status: String? = null,
    val content: List<ContentBlock>? = null,
    val metadata: Map<String, String>? = null,
    val createdAt: Int? = null,
    val threadId: String? = null,
    val incompleteDetails: Any? = null,
    val completedAt: Int? = null,
    val incompleteAt: Int? = null,
    val assistantId: String? = null,
    val runId: String? = null,
    val fileIds: List<String>? = null
)
